src/routers/datatypeRouters/edges/regulatory_regions_genes.ts

```diff
@@ -25,6 +25,17 @@ const edgeSources = z.object({
   ]).optional()
 })
 
+const regulatoryRegionType = z.enum([
+  'candidate_cis_regulatory_element',
+  'enhancer',
+  'CRISPR_tested_element'
+])
+
+const biochemicalActivity = z.enum([
```

Comment:
biochemicalActivity for regulatory region is already defined in regulatory_regions.ts.

Reply:
yeah the issue is not all regulatory_regions have edges to genes, so I redefined biochemicalActivity enum list here, which is a subset from the one in regulatory_regions.ts.